portfolio-project-matching-app/components/ProjectCard.js

```diff
@@ -0,0 +1,44 @@
+import Button from '../components/Button'
+import UserIcon from '../components/UserIcon'
+import { doc } from '@firebase/firestore'
+
+const ProjectCard = ({ project }) => {
+    const applyToProject = async (e) => {
```

Comment:
Wanted to get our DB "schema" figured out before I went crazy integrating things with the DB.